packages/aws-cdk/test/integ/run-against-dist.bash

```diff
@@ -52,6 +52,12 @@ function serve_npm_packages() {
       package_names="${package_names} ${package}"
     done
 
+    # manually add cdk-assets since its not prefixed with @aws-cdk and 
+    # hence isn't picked up from package.json
+    echo "Adding cdk-assets to CLI dependencies"
+    tarballs="${tarballs} cdk-assets-${version}.tgz"
+    package_names="${package_names} cdk-assets"
+
```

Comment:
doesn't it make sense to put all the tarballs and opt out of what you don't need instead of opt-in?